llvm/lib/Target/RISCV/MCTargetDesc/RISCVBaseInfo.h

```diff
@@ -331,6 +331,9 @@ enum OperandType : unsigned {
   OPERAND_RVKRNUM_2_14,
   OPERAND_SPIMM,
   OPERAND_LAST_RISCV_IMM = OPERAND_SPIMM,
+  // Operand is a 3-bit rounding mode, '111' indicates FRM register.
+  // Represents 'frm' argument passing to floating-point operations.
+  OPERAND_FRMARG,
```

Comment:
I extracted this and committed separately. 13a3c4f

Reply:
Addressed